javatests/arcs/core/entity/HandleManagerCloseTest.kt

```diff
@@ -43,35 +47,44 @@ class HandleManagerCloseTest {
     fun setup() {
         DriverAndKeyConfigurator.configure(null)
         SchemaRegistry.register(Person)
-        SchemaRegistry.register(Hat)
+        SchemaRegistry.register(HandleManagerTestBase.Hat)
     }
 
     fun createHandleManager() = EntityHandleManager("testArc", "", FakeTime(), scheduler)
 
     @Test
-    fun closehandleManagerStopUpdates() = runBlockingTest {
+    fun closehandleManagerStopUpdates() = runBlocking {
```

Comment:
is this change to runBlocking because the timeouts fail fast with runBlockingTest?

Reply:
Not the timeouts, but the fact that the scheduler launches jobs on other dispatchers -and runBlockingTest is bad about supporting that.